Geofencer/Geofencer/SavedPlaces.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  File.swift
+//  Geofencer
+//
+//  Created by Brett Parsekian on 8/6/19.
+//  Copyright © 2019 bparsekian. All rights reserved.
+//
+
+import Foundation
+
+struct SavedPlaces {
+    var places: [Place]
+}
+
+struct Place: Decodable {
+    let title: String
+    let coordinates: GeoCoordinates
+}
+
+struct GeoCoordinates {
+    var latitude : Double
+    var longitude: Double
+}
+
+extension GeoCoordinates : Decodable {
+    private enum CodingKeys: String, CodingKey {
+        case latitude = "Latitude"
+        case longitude = "Longitude"
+    }
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let latitudeString = try container.decode(String.self, forKey: .latitude)
+        let longitudeString = try container.decode(String.self, forKey: .longitude)
+        latitude = Double(latitudeString) ?? 0
+        longitude = Double(longitudeString) ?? 0
+    }
+}
+
+struct Location : Decodable {
+    let geoCoordinates : GeoCoordinates
+    
+    private enum CodingKeys: String, CodingKey {
+        case geoCoordinates = "Geo Coordinates"
+        case latitude = "Latitude"
+        case longitude = "Longitude"
+    }
+    
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let gCoordinates = try container.decodeIfPresent(GeoCoordinates.self, forKey: .geoCoordinates)
+        if (gCoordinates == nil) {
```

Comment:
I'm not too picky about this myself, but it is actually a bit strange to see these parentheses around the conditional here.  It's more typical in Swift codebases not to include them in if-statements and other conditional blocks